sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TextAnalyticsAsyncClientTest.java

```diff
@@ -2612,7 +2612,6 @@ public void analyzeExtractSummaryActionMaxSentenceCountInvalidRangeException(Htt
         }
```

Comment:
maybe add a test that ensures that if fhirVersion is passed, then fhirBundle is returned?

Reply:
updated